pages/api/auth/[...nextauth].js

```diff
@@ -36,6 +38,16 @@ function getCallbacks (req) {
         token.sub = Number(token.id)
       }
 
+      // response is only defined during signup/login
+      if (req && res) {
+        req = new NodeNextRequest(req)
+        res = new NodeNextResponse(res)
+        const secret = process.env.NEXTAUTH_SECRET
+        const jwt = await encodeJWT({ token, secret })
+        const me = await prisma.user.findUnique({ where: { id: token.id } })
+        setMultiAuthCookies(req, res, { ...me, jwt })
+      }
```

Comment:
Can you confirm this doesn't run on every request? We don't want every request to incur an additional roundtrip to the database.

Reply:
Yes, it's only run during login or signup.